src/charm.py

```diff
@@ -127,24 +164,32 @@ def _on_gunicorn_pebble_ready(self, event: ops.framework.EventBase) -> None:
 
     def _configure_workload(self, event: ops.charm.EventBase) -> None:
         """Configure the workload container."""
-        pebble_config = self._get_pebble_config(event)
+        pebble_config = self._get_gunicorn_pebble_config(event)
```

Comment:
I think this should be renamed to `gunicorn_pebble_config = self.get_gunicorn_pebble_config(event)` for clarity